poc/mastic.py

```diff
@@ -337,6 +338,35 @@ def unshard(self,
             agg_result.append(self.flp.decode(chunk[1:], meas_count))
         return agg_result
 
+    def encode_agg_param(self, agg_param: MasticAggParam) -> bytes:
+        level, prefixes, do_weight_check = agg_param
+        if level not in range(2 ** 16):
+            raise ValueError('level out of range')
+        if len(prefixes) not in range(2 ** 32):
+            raise ValueError('number of prefixes out of range')
+        encoded = bytes()
+        encoded += to_be_bytes(level, 2)
+        encoded += to_be_bytes(len(prefixes), 4)
+        # NOTE: The do_weight_check is the only difference between Mastic's and
+        # Poplar1's `encode_agg_param``
+        encoded += to_be_bytes(do_weight_check, 1)
```

Comment:
nit: Python seems to be alright with an implicit cast here, but this makes me shutter
```suggestion
        encoded += to_be_bytes(int(do_weight_check), 1)
```